### PR TITLE
partners 2014

### PR DESCRIPTION
**Pull request by davojta**
adding partners in format suitable for cms page for cpm2014 from https://trello.com/c/gUxjsR7E/26-fixture-cpm2014-partners

### CODE REVIEW

File: data/raw/cpm2014_partners.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    "name_be": "Nomadic Film Club",
+    "name_ru": "Nomadic Film Club",
+    "name_en": "Nomadic Film Club",
+    "link": "http://vagrant.kinaklub.org/",
+    "image": "images/nomadic.png"
+  },
+  {
+    "name_be": "Минская Киношкола-студия",
+    "name_ru": "Минская Киношкола-студия",
+    "name_en": "Минская Киношкола-студия",
```
Comment (abitrolly) [inline data/raw/cpm2014_partners.json:33]: That's not translated. Is there a strategy to get this done? Should these be left blank then?

Comment (davojta) [inline data/raw/cpm2014_partners.json:33]: it's for 1st milestone to release it untranslated.
After 1st milestone we will collect all untranslated text and will send them to translation coordinator for translate.

These data was always only on 1 lang.

**Comment by abitrolly** [comment]
I can LGTM this as is, but better if I spend some time to review that everything is ok.

**Comment by abitrolly** [comment]
`relax450.png` and `where450.png` - are they used anywhere?

**Comment by abitrolly** [comment]
`100second.png` usage is also questionable. JSON content LGTM.

**Comment by davojta** [comment]
These images will be used for other years, i can recheck them to exclude
used for these year

**Comment by abitrolly** [comment]
Yes, it is better to add only images that are used to keep track of them.

**Comment by abitrolly** [comment]
@davojta extra images, see #43 for `datacheck.py` script:

```
C:\next.filmfest.by\data>py "datacheck.py"
.\raw\cpm2014_partners.json
  entries: 18
    error - extra 'raw\images\100second.png'
    error - extra 'raw\images\cinewest.png'
    error - extra 'raw\images\eksperiment.png'
    error - extra 'raw\images\relax450.png'
    error - extra 'raw\images\videomaker.png'
    error - extra 'raw\images\where450.png'
```

**Comment by davojta** [comment]
I plan rework this pr to make it more consistent. I will add data year by
year to repo from 2012 to 2016 in separate pr

**Comment by abitrolly** [comment]
Let's just remove extra files and commit it. It is already reviewed.

**Comment by abitrolly** [comment]
In other words, let's apply "progressive enhancement" methodology.

**Comment by davojta** [comment]
Okey, will try finish it soon)

**Comment by abitrolly** [comment]
@davojta I can remove them for you and merge. Deal?

**Comment by davojta** [comment]
Okey, you can do it. i can do it tommorow morning)

**Comment by abitrolly** [comment]
My ningitsu > yours, so I'll do it faster. )

**Comment by abitrolly** [comment]
OMG, I screwed up rebase,

**Comment by abitrolly** [comment]
Good job, @davojta. =)

**Pull request merged**